setup.py

```diff
@@ -340,11 +338,45 @@ def run(self):
 
         if not has_errors:
             print('...PASSED')
+
+        all_playbooks, included_playbooks = find_playbooks()
+
+        print('#' * 60)
+        print('Invalid Playbook Include Checks')
+        invalid_include = []
+        for playbook in sorted(included_playbooks):
```

Comment:
No need to sort a set.

Reply:
Was sorting for ordered output.  Move the sort to the resultant list if it exists.